docs/user-guide/ze-working-with-data-sets.md

```diff
@@ -152,52 +216,50 @@
 
```

Comment:
For the note "You can delete a data set before you delete its members.", I would change this to "Deleting a partitioned data set will also delete its members" or something similar.

I think this note is trying to tell users that they do not have to manually delete all of a PDS's members before deleting the PDS itself. It might make sense to remove the note altogether.